qt/scientific_interfaces/ISISReflectometry/GUI/Preview/QtPreviewView.cpp

```diff
@@ -21,16 +21,19 @@
 using namespace Mantid::Kernel;
 
 namespace MantidQt::CustomInterfaces::ISISReflectometry {
-QtPreviewView::QtPreviewView(QWidget *parent) : QWidget(parent) {
+QtPreviewView::QtPreviewView(QWidget *parent)
+    : QWidget(parent), m_imageInfo(new MantidQt::MantidWidgets::ImageInfoWidgetMini(this)) {
```

Comment:
Just double checking that this is correct to be creating a raw pointer with `new`? I think so because as it is a QWidget then Qt will destroy it when the parent is destroyed - is that your understanding too?

Reply:
I think so, yes. That's the way we've handled adding QWidgets in other parts of the GUI, so I've just followed that pattern